docs/cli.rst

```diff
@@ -0,0 +1,45 @@
+Command Line Interface
+======================
+
+Nameko ships with a command line interface to make hosting and interacting with services as easy as possible.
+
+Running a Service
+-----------------
+
+::
+
+    $ nameko run [<module>[:<ServiceClass>]]
```

Comment:
Ah, wishful thinking. I'd like to change the discovery mechanism so this would be valid. Will make it right for now though